CHANGELOG.md

```diff
@@ -1,3 +1,158 @@
+# 7 (UNRELEASED)
```

Comment:
I don't think this should be 7, we should probably use helix version to avoid confusion.